Automattic/wp-calypso
# State: Refactor `getFileUploader` away from `redux-bridge`

#### Changes proposed in this Pull Request

@jsnajdr kindly reminded me in https://github.com/Automattic/wp-calypso/pull/53184#commitcomment-51330039 that `redux-bridge` was created solely to help with flux migration, and we should be going away from it, rather than introducing new usages.

This PR refactors away from `redux-bridge` in the `getFileUploader()` thunk. The refactor is very straightforward, as the function only needs the `getState()` part of the store.

While investigating and testing this, I discovered that the `addWoocommerceProductImage` thunk has been unused since #51012 landed. I decided to use the opportunity and remove it here.

#### Testing instructions

* Go to the media section and verify uploading files still works.
* Verify `addWoocommerceProductImage` is indeed unused.
* Verify tests still pass.
